raftwal/log.go

```diff
@@ -346,7 +346,7 @@ func (lf *logFile) bootstrap() error {
 		return y.Wrapf(err, "Error while creating base IV, while creating logfile")
 	}
 	// Initialize base IV.
-	lf.baseIV = make([]byte, 8)
-	y.AssertTrue(8 == copy(lf.baseIV, buf[8:]))
+	lf.baseIV = y.Copy(buf[8:])
+	y.AssertTrue(len(lf.baseIV) == 8)
```

Comment:
These lines were conflicting while merge. I opted for `y.Copy()` as better code practice to use existing functions. Also, accepted changes were made later in `master`.